spec/Model.spec.js

```diff
@@ -765,9 +799,9 @@ describe('Model', () => {
         description: 'Example description'
       })
 
-      expect(note.dirtyAttributes).toEqual([])
+      expect(note.dirtyRelationships).toEqual([])
       note.todo = todo
-      expect(note.dirtyAttributes).toEqual(['relationships.todo'])
+      expect(note.dirtyRelationships).toEqual(['relationships.todo'])
```

Comment:
We could change this to just return `todo` instead of `relationships.todo` 
Does anyone have a preference?